frontend/src/components/Menus/MainMenu/Menu.js

```diff
@@ -108,13 +109,22 @@ export const Menu = ({ expanded, clickHandler }) => {
     { text: 'Collaborations', path: '/collaborations', icon: CollaborationsIcon },
     {}, // an empty object causes a Divider component to render in the menu
     { text: 'TIN Dashboard', path: 'https://redcap.vanderbilt.edu/plugins/TIN/', icon: TINLogoIcon },
-    {
-      text: 'HEAL Dashboard',
-      path: 'https://redcap.vanderbilt.edu/plugins/TIN/sso/send_login?target-url=https://heal-ctmd.renci.org/api/auth',
-      icon: HEALIcon,
-    },
   ]
 
+  if (process.env.REACT_APP_IS_HEAL_SERVER === 'true') {
+    menuItems.push({
+      text: 'CTMD',
+      path: 'https://ctmd.renci.org/',
+      icon: CTMDIcon,
+    })
+  } else {
+    menuItems.push({
+      text: 'HEAL CTMD',
+      path: 'https://heal-ctmd.renci.org/',
+      icon: HEALIcon,
+    })
```

Comment:
@mbwatson It looks to me that if and else actions are reversed here, i.e., line 115 to 119 should apply when REACT_APP_IS_HEAL_SERVER env var is NOT true, right?

Reply:
woops! thanks for catching